lms/djangoapps/certificates/views.py

```diff
@@ -481,9 +468,19 @@ def render_html_view(request, user_id, course_id):
     configuration = CertificateHtmlViewConfiguration.get_config()
     context.update(configuration.get('default', {}))
 
+    # Translators:  This is the copyright line which appears at the bottom of the certificate page/screen
+    context['copyright_text'] = _('&copy; {year} {platform_name}. All rights reserved.').format(
```

Comment:
The people who do translations into other languages are not necessarily technical people, and probably have no idea what `&copy;` means or what to do with it. Also, I can't see the order of the first few items in the list ever changing from language to language. I'd suggest changing this line to:
```python
context['copyright_text'] = '&copy; {year} {platform_name} '.format(...) + _("All rights reserved.")
```
@sarina, do you agree? Or is this a terrible suggestion?